DOCS/man/osc.rst

```diff
@@ -167,9 +167,8 @@ Configurable Options
 ``seekbarstyle``
     Default: bar
 
-    Sets the style of the seekbar, slider (diamond marker), knob (circle
-    marker with guide), or bar (fill).
-    Default pre-0.21.0 was 'slider'.
+    Sets the style of the playback position marker and overall shape
+    of the seekbar: ``bar``, ``diamond`` or ``knob``.
```

Comment:
Why rename `slider` to `diamond`?

Reply:
Because both diamond and knob can be sliders now. It's part of the seekrangestyle option now.

Reply:
Right, makes sense.